adapters/flipp/flipp.go

```diff
@@ -230,7 +243,22 @@ func buildBid(decision *InlineModel, impId string) *openrtb2.Bid {
 		if decision.Contents[0].Data.Width != 0 {
 			bid.W = decision.Contents[0].Data.Width
 		}
-		bid.H = 0
+		customDataInterface := decision.Contents[0].Data.CustomData
+		customDataMap := customDataInterface.(map[string]interface{})
+
+		if flippExtParams.Options.StartCompact {
+			defaultHeight = defaultCompactHeight
```

Comment:
Why do we need this extra `defaultHeight` variable. Can't we just assign this directly to `bid.H`?